app/Http/routes.php

```diff
@@ -25,7 +25,17 @@
 ], function () {
     Route::get('example', 'ExampleController@get');
     Route::post('example', 'ExampleController@post');
-});
+
+    #add by peng@team4
+    Route::get('inventory','InventoryController1@index');
+    Route::post('inventory','InventoryController1@store');
+    Route::get('inventory/{tag}','InventoryController2@show');
+    Route::put('inventory/{tag}','InventoryController2@update');
+    Route::delete('inventory/{tag}','InventoryController2@destroy');
+
+    Route::get('inventory-types','ItemTypeController@index');
```

Comment:
The inventory types collection should have URL `inventory/types`. Think of it as a "subdirectory" of inventory.